requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy>=1.19.3
 opencv-python>=4.4.0.46
 pyfakewebcam>=0.1.0
-mediapipe==0.8.5
+mediapipe==0.8.6.2
```

Comment:
Do you experience issue https://github.com/fangfufu/Linux-Fake-Background-Webcam/issues/127?

Reply:
I don't - I've only used it a couple days, but so far i've been able to leave it on continuously without issue. I'm on Mint 19.3, and i had to compile v4l2 manually (not using the dkms from apt) to get this project to work, so it's possible it's just different setups.

Reply:
It seems to work for me so far. I am going to leave it running for a bit before deciding what to do.